cmake/configs/default.cmake

```diff
@@ -66,6 +66,7 @@ hunter_config(Libssh2 VERSION 1.7.0)
 hunter_config(Lua VERSION 5.3.2)
 hunter_config(MySQL-client VERSION 6.1.6)
 hunter_config(OpenBLAS VERSION 0.2.19-p0)
+hunter_config(OpenCL VERSION 2.1-p0)
```

Comment:
I was a bit confused about how the test were being run without this, guess a little reading pays off. I am assuming example code needs to be removed as well?